test/00/deleteKey.sh

```diff
@@ -37,10 +37,11 @@ proc afterMinskyStarted {} {
   set numWires [model.numWires]
   focus .wiring.canvas
 
-  set w [minsky.model.wires.@elem 0]
-  set x {[\$w.coords.@elem 0]}
-  set y {[\$w.coords.@elem 1]}
-  deleteKey \$x \$y
+#  set w [minsky.model.wires.@elem 1]
+#  set x {[\$w.coords.@elem 0]}
+#  set y {[\$w.coords.@elem 1]}
+#  deleteKey \$x \$y
+  deleteKey 404 265
```

Comment:
Yes - but this makes the test more fragile again. What about reinstating the previous TCL code, but setting x and y to the midpoints:
set x [expr 0.5*[\$w.coords.@elem 0]+[\$w.coords.@elem 2]]
set Y [expr 0.5*[\$w.coords.@elem 1]+[\$w.coords.@elem 3]]

Reply:
Thanks for the suggestion, it hadn't occurred to me. I've tried it for element 0 and some others, but it has not yet worked for me. If I succeed I will put in a commit. I don't understand why curly braces are required around these expressions?

  set x {[expr 0.5*[[\$w.coords.@elem 0]+[\$w.coords.@elem 2]]]}
  set y {[expr 0.5*[[\$w.coords.@elem 1]+[\$w.coords.@elem 3]]]}

They aren't in other tcl files where coordinate variables are set.